save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestExecutionController.kt

```diff
@@ -100,6 +101,25 @@ class TestExecutionController(private val testExecutionService: TestExecutionSer
                 }
             }
 
+    /**
+     * @param executionId an ID of Execution to group TestExecutions
+     * @param authentication
+     * @return a list of [TestExecutionDto]s
+     */
+    @GetMapping(path = ["/api/$v1/testLatestExecutionsPieChart"])
```

Comment:
I think it's too specific to have it in common API. You could have a method `/testLatestExecutionsStatistics`, that returns a map `test suite name` -> `number of tests` or something similar. Then it can be reused not only on our frontend, but in any integration. Also color is needed only on frontend, so there is no need to calculate it on backend.